extra/pipsize.py

```diff
@@ -0,0 +1,33 @@
+import os
+import pkg_resources
+from tabulate import tabulate
```

Comment:
needs to be added to dependencies somewhere. I'd propose a `requirements-dev.txt` file for things like this.